components/channel_header/components/__snapshots__/header_icon_wrapper.test.js.snap

```diff
@@ -15,7 +15,7 @@ exports[`components/channel_header/components/HeaderIconWrapper should match sna
         placement="right"
       >
         <FormattedMessage
-          defaultMessage="Flagged Posts"
+          defaultMessage="See flagged posts"
```

Comment:
0/5 but keeping tooltips short is valuable in my mind

Reply:
Shortened it back to 'Flagged posts'.

Reply:
@AninditaBasu This PR has been merged but I'm confused what was agreed on this comment.  You've mentioned reverting to short tooltips but I'm seeing different here and on what were merged into master. Maybe we can still fix in https://github.com/mattermost/mattermost-webapp/pull/4803 once you update it?